### PR TITLE
sink(ticdc): omit empty field in tidbExtension

### PR DESCRIPTION
**Pull request by hi-rustin**
### What problem does this PR solve? 
omit empty field in tidbExtension. 

### What is changed and how it works?
omit empty field in tidbExtension:
1. DDL or DML event no watermark ts
2. Checkpoint event no commit ts

### Check List 

Tests 

 - Unit test

Code changes

None

Side effects

None

Related changes

None

### Release note 

```release-note
None
```

### CODE REVIEW

**Comment by hi-rustin** [comment]
/cc @3AceShowHand @overvenus 
Could you please take a look? Thanks! 💚 💙 💜 💛 ❤️

File: cdc/sink/codec/canal_flat.go [hunk 1]
```diff
@@ -143,8 +143,8 @@ func (c *canalFlatMessage) getMySQLType() map[string]string {
 }
 
 type tidbExtension struct {
-	CommitTs    uint64 `json:"commit-ts"`
-	WatermarkTs uint64 `json:"watermark-ts"`
+	CommitTs    uint64 `json:"commit-ts,omitempty"`
```
Comment (overvenus) [inline cdc/sink/codec/canal_flat.go:146]: Could you rename `"commit-ts"` to `commitTs` ? Since canal-json use camelCase style.

Comment (hi-rustin) [reply cdc/sink/codec/canal_flat.go:146]: Good catch!

Comment (hi-rustin) [reply cdc/sink/codec/canal_flat.go]: Addressed!

**Review by overvenus — APPROVED** [review]
:shipit:

**Pull request merged**